src/check-update.ts

```diff
@@ -0,0 +1,250 @@
+import { BUILDER_TAG, HOOKS_TAG, RUNTIME_TAG } from "./mod.ts";
+
+const DENO_SLACK_SDK = "deno_slack_sdk";
+const DENO_SLACK_API = "deno_slack_api";
+const DENO_SLACK_HOOKS = HOOKS_TAG.substring(0, HOOKS_TAG.indexOf("@"));
+const DENO_SLACK_BUILDER = BUILDER_TAG.substring(0, BUILDER_TAG.indexOf("@"));
+const DENO_SLACK_RUNTIME = RUNTIME_TAG.substring(0, RUNTIME_TAG.indexOf("@"));
+const IMPORT_MAP_SDKS = [DENO_SLACK_SDK, DENO_SLACK_API];
+const SLACK_JSON_SDKS = [
+  DENO_SLACK_HOOKS, // should be the only one needed once the get-scripts hook is supported
+  DENO_SLACK_BUILDER, // but just in case, we can look for builder and runtime as well!
+  DENO_SLACK_RUNTIME,
+];
+
+interface UpdateResponse {
+  update: boolean;
+  breaking: boolean;
+  message: string;
+  error: {
+    message: string;
+  };
```

Comment:
This was originally just a string, but after discussing with @mwbrooks, this leaves open the possibility of including more information at a later date (`remediation`, `code`).